### DISCUSSION TITLE
Do something about packages that install fine on real system but for some reason are not installing in WinGetSvc-Validation bot environment

### DISCUSSION THREAD

**Discussion by soredake** [microsoft/winget-pkgs#158537] [General]
There are some packages that install fine on real system but fail in WinGetSvc-Validation bot environment which results in inability to merge pr for such software, list of such software that I found:
https://github.com/microsoft/winget-pkgs/pull/149994
https://github.com/microsoft/winget-pkgs/pull/147376
https://github.com/microsoft/winget-pkgs/issues/157344

**Comment by schembriaiden** [comment]
These are usually verified manually, try mentioning [at sign]stephengillie and he will help.

**Reply by soredake** [reply]
I know about this, thanks, but this really needs to be solved, pinging them every time each of those software updates is not really a long-term solution.

**Reply by schembriaiden** [reply]
I totally get it and I agree with you, my biggest issue is that it doesn't tell you exactly what's wrong with the install, it only tells you that it fails, and even when you go to Azure, no log of an error. This sadly increases pressure on the maintainers like stephengillie, etc...
